SpiceSharp/Components/RLC/RES/BaseParameters.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SpiceSharp.Attributes;
+
+namespace SpiceSharp.Components.ResistorBehaviors
+{
+    /// <summary>
+    /// Base set of parameters for a <see cref="Resistor"/>
+    /// </summary>
+    public class BaseParameters : ParameterSet
+    {
+        /// <summary>
+        /// Parameters
+        /// </summary>
+        [PropertyName("resistance"), PropertyInfo("Resistance", IsPrincipal = true)]
+        public Parameter Resistance { get; } = new Parameter();
+        [PropertyName("temp"), PropertyInfo("Instance operating temperature", Interesting = false)]
+        public double TemperatureCelsius
+        {
+            get => Temperature - Circuit.CelsiusKelvin;
+            set => Temperature.Set(value + Circuit.CelsiusKelvin);
+        }
+        public Parameter Temperature { get; } = new Parameter(300.15);
```

Comment:
@svenboulanger Please use const instead of 300.15

Reply:
I don't really think it'll be practical to make constants for all parameters with any default parameter that is not 0 by default.

Reply:
@svenboulanger ok